Dockerfile

```diff
@@ -17,9 +17,11 @@ RUN apt-get update \
    libfontconfig1 \
  && apt-get clean autoclean \
  && apt-get autoremove \
- && rm -rf /var/lib/{apt,dpkg,cache,log}
+ && rm -rf /var/lib/{apt,dpkg,cache,log} \
+ && mkdir -p /cache /config /media \
+ && chmod 777 /cache /config /media
```

Comment:
Is 755 not sufficient?

Reply:
No, the user creating these folders is always root. The user running jellyfin can change depending on how one runs the container.

Reply:
Shouldn't they be chown'd then?

Reply:
We can't chown at this point since we don't know the user. We can't chown after they've been mounted since that would carry over to the host volume.

Reply:
I see